lib/cli/render-help/general.js

```diff
@@ -3,31 +3,32 @@ import globalOptions from '../commands-options-schema.js';
 import generateCommandUsage from './generate-command-usage.js';
 import renderOptions from './options.js';
 
-const { writeText, style } = utils;
+const { writeText, style, log } = utils;
 
 export default ({ loadedPlugins, commandsSchema, version }) => {
-  writeText(
-    `Serverless Framework v${version}`,
-    null,
-    style.aside('Usage'),
-    'serverless <command> <options>',
-    'sls <command> <options>',
-    null,
-    style.aside('Get started'),
-    `Run ${style.strong('serverless')} to interactively setup a project.`,
-    null,
-    style.aside('Dashboard'),
-    'See metrics, traces, logs, errors, deployments and more in Serverless Framework Dashboard.',
-    `Learn more: ${style.linkStrong('https://app.serverless.com')}`,
-    null,
-    style.aside('Plugins'),
-    'Extend the Serverless Framework with plugins.',
-    `Explore plugins: ${style.linkStrong('https://serverless.com/plugins')}`,
-    null,
-    style.aside('Options')
-  );
+  // const header = (txt) => style.strong(style.title(txt));
```

Comment:
Can we remove these comments @eahefnawy or are they part of the renderer? It is kind of hard to tell with the way it is being highlighted.

Reply:
Ah sorry yeah I was experimenting and forgot to remove it.